### PR TITLE
Live Demo register page description alignment.

### PR DESCRIPTION
**Pull request by denizdemirkan**
Resolves https://github.com/volosoft/vs-internal/issues/3434

This branch improving UI alignment in Live Demo register page.

new
new2

### CODE REVIEW

**Comment by gizemmutukurt** [comment]
I am getting related errors while testing

image

File: abp_io/AbpIoLocalization/AbpIoLocalization/Commercial/Localization/Resources/es.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "StartUsingYourLicenseNow": "¡Empiece a usar su licencia ahora!",
     "WelcomePage": "Pagina de bienvenida",
     "UnsubscriptionExpireEmail": "Cancelar la suscripción a los correos electrónicos de recordatorio de la fecha de vencimiento de la licencia",
-    "UnsubscribeLicenseExpireEmailReminderMessage": "Esta suscripción de correo electrónico solo contiene un recordatorio de la fecha de vencimiento de su licencia.",
+    "UnsubscribeLicenseExpireEmailReminderMessage": "Esta suscripción de correo electrónico solo contiene un recordatorio de la fcha de vencimiento de su licencia.",
```
Comment (salihozkara) [inline abp_io/AbpIoLocalization/AbpIoLocalization/Commercial/Localization/Resources/es.json]: Please revert this line

**Review by salihozkara — APPROVED** [review]

**Comment by oykuermann** [comment]
I got errors when trying to run Account.Web project:
image

**Comment by denizdemirkan** [comment]
Fixed https://github.com/abpframework/abp/pull/18633#issuecomment-1875040280

@oykuermann 
To test this issue
1 - change branch on **abp** repository to **issue-3434** (others must be on dev branch)
2 - Run Account app
3 - Run Commercial app
4 - Sign-out if you are logged in
5 - Go to Live Demo page at commercial domain.

**Comment by gizemmutukurt** [comment]
You do not need an Account app to go to this page. Running Commercial is enough.

**Review by gizemmutukurt — APPROVED** [review]

**Pull request merged**